src/jobs/models/jobManager.ts

```diff
@@ -151,6 +143,13 @@ export class JobManager {
     return hasPendingTasks && (job.status === OperationStatus.PENDING || job.status === OperationStatus.IN_PROGRESS);
   }
 
+  public isResettableJob(job: IGetJobResponse): IIsResettableResponse {
+    return {
+      jobId: job.id,
+      isResettable: !job.isCleaned && (job.status === OperationStatus.FAILED || job.status === OperationStatus.SUSPENDED),
```

Comment:
Place to reuse previously defined RESETABLE_STATUSES

Reply:
I will open an issue on this refactor